README.md

```diff
@@ -36,9 +36,50 @@ var getGlobal = function () {
 
 ## HTML and the `WindowProxy`
 
-In HTML 5, the global object is separated into the `Window` and the `WindowProxy`. New attributes are set on the `Window`, but top-level this has the identity of the `WindowProxy`. The `WindowProxy` forwards all object operations to the underlying `Window`, but as the page changes, the global object maintains the same identity while the underlying `Window` is swapped out.
```

Comment:
Side note: [it's "HTML5", not "HTML 5"](https://blog.whatwg.org/spelling-html5). But it's 2019 so let's just call it "HTML".